src/ml/neural_net/mps_layer_conv_padding.m

```diff
@@ -0,0 +1,57 @@
+#include <ml/neural_net/mps_layer_conv_padding.h>
+
+@implementation ConvolutionPadding
+
++ (BOOL) supportsSecureCoding {
+  return true;
+}
+
+- (id) initWithParams:(int)paddingWidth
+        paddingHeight:(int)paddingHeight
+          strideWidth:(int)strideWidth
+         strideHeight:(int)strideHeight {
+  @autoreleasepool {
+    self = [self init];
+    mPaddingWidth = paddingWidth;
```

Comment:
The idiomatic implementation of this initializer would be something like:
```
self = [super init];
if (self) {
  _paddingHeight = paddingHeight;
  ....
}
return self;
```

Reply:
done!